### PR TITLE
Translate CVE-2023-28755 (ko)

### PR DESCRIPTION
**Pull request by marocchino**
you can found original text from
https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md

### CODE REVIEW

File: ko/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+ReDoS 취약점에 대한 보안 수정이 포함된 uri gem 버전 0.10.0.1, 0.10.2, 0.11.1 및 0.12.1을 릴리스했습니다. 이 
+취약점에는 CVE 식별자 [CVE-2023-28755](https://nvd.nist.gov/vuln/detail/CVE-2023-28755)가 할당되었습니다.
+
+## 세부 내용
+
+URI 구성 요소에서 ReDoS 문제가 발견되었습니다. URI 구문 분석기가 특정 문자가 포함된 유효하지 않은 URL을 잘못 처리합니다. 이로 인해 URI 객체에 대한 문자열 구문 분석 실행 시간이 증가합니다.
+
+`uri` gem의 0.10.1, 0.10.2, 0.11.0, 0.12.0 및 모든 0.10.0 이하 버전이 이 취약점에 취약합니다.
```
Comment (marocchino) [inline ko/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md:18]: isn't this typo? I think 0.10.2 is patched version.

**Comment by yous** [comment]
Ref #3006.

File: ko/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md (hunk 1 above)
Comment (marocchino) [reply ko/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md:18]: #3008

**Comment by yous** [comment]
Let's wait for #3009.

**Comment by yous** [comment]
> Let's wait for #3009.

Now we can apply the update.

File: ko/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## 세부 내용
+
+URI 구성 요소에서 ReDoS 문제가 발견되었습니다. URI 구문 분석기가 특정 문자가 포함된 유효하지 않은 URL을 잘못 처리합니다. 이로 인해 URI 객체에 대한 문자열 구문 분석 실행 시간이 증가합니다.
+
+`uri` gem의 0.12.0, 0.11.0, 0.10.1, 0.10.0과 모든 0.10.0 이하 버전이 이 취약점에 취약합니다.
+
+## 권장 조치
+
+`uri` gem을 0.12.1로 업데이트하는 것이 좋습니다. 이전 Ruby 시리즈에서는 포함된 버전과의 호환성을 보장하기 위해 다음과 같이 업데이트할 수 있습니다.
```
Comment (yous) [inline ko/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md:22]:
```suggestion
`uri` gem을 0.12.1로 업데이트하는 것이 좋습니다. 이전 Ruby 버전대에 포함된 버전과의 호환성을 보장하기 위해 다음과 같이 업데이트할 수 있습니다.
```

File: ko/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+* Ruby 2.7: `uri` 0.10.0.1로 업데이트
+* Ruby 3.0: `uri` 0.10.2로 업데이트
+* Ruby 3.1: `uri` 0.11.1로 업데이트
+* Ruby 3.2: `uri` 0.12.1로 업데이트
+
+`gem update uri`를 사용하여 업데이트할 수 있습니다. 번들러를 사용하는 경우 `gem "uri", ">= 0.12.1"`(또는 위에 언급된 다른 버전)을 `Gemfile`에 추가하세요.
```
Comment (yous) [inline ko/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md:29]:
```suggestion
`gem update uri`를 사용하여 업데이트할 수 있습니다. bundler를 사용하는 경우 `gem "uri", ">= 0.12.1"`(또는 위에 언급된 다른 버전)을 `Gemfile`에 추가하세요.
```

**Review by yous — APPROVED** [review]

**Pull request merged**